packages/vkui/src/components/Tabbar/Tabbar.e2e.tsx

```diff
@@ -5,6 +5,7 @@ import { TabbarPlayground } from './Tabbar.e2e-playground';
 
 test.use({
   onlyForPlatforms: [Platform.ANDROID, Platform.IOS],
+  threshold: 0,
```

Comment:
```suggestion
  toMatchSnapshot: {
    threshold: 0,
  }
```
мб такую структуру сделаем? чтобы было поднято к чему отсылается